Patches/FFmpeg/4.4.1/mpegtsenc.c

```diff
@@ -114,6 +114,7 @@ typedef struct MpegTSWrite {
     int64_t sdt_period_us;
     int64_t last_pat_ts;
     int64_t last_sdt_ts;
+    int omit_sdt;
```

Comment:
Is there a ffmpeg specific reason this can't be a `bool`?

Reply:
FFmpeg seems to prefer `int`s over `bool`s (see the next member here: `int omit_video_pes_length`). Could be for memory alignment, or to leave space for future expansion of options, or because FFmpeg is old and C89 didn't have a native `bool` type. Whatever the reason, I figured it's best to stick with the surrounding coding style.